scripts/dev

```diff
@@ -138,6 +138,24 @@ namespace :db do
   task :drop_connections do
     `echo "select pg_terminate_backend(pg_stat_activity.pid) from pg_stat_activity where datname='postgres' AND state='idle'" | psql`
   end
+
```

Comment:
`flyway clean` exists, but it runs all the migrations and takes many seconds. `scripts/dev db:clean` runs nearly instantly since there is usually very little data in these tables.